src/components/AnnotationsModule.vue

```diff
@@ -52,3 +46,15 @@ export default defineComponent({
   display: block;
 }
 </style>
+
+<style>
+.annotation-header {
```

Comment:
I would recommend moving this out into its own css stylesheet in `components/styles` and doing a `<style scoped src="..." />` to keep names scoped.